app/services/data_migrations/end_of_cycle_cancel_outstanding_references.rb

```diff
@@ -0,0 +1,35 @@
+module DataMigrations
+  class EndOfCycleCancelOutstandingReferences
+    TIMESTAMP = 20220913163416
+    MANUAL_RUN = true
+    RECRUITMENT_CYCLE_YEAR = 2022
+    PHASE = 'apply_1'.freeze
+
+    def change
+      records.each do |record|
+        record.update!(
+          feedback_status: :cancelled_at_end_of_cycle,
+          cancelled_at: Time.zone.now,
+        )
+      end
+    end
+
+    def dry_run
+      "The total of #{records.count} references will be cancelled"
+    end
+
+    def records
+      ApplicationReference.joins(:application_form)
+        .feedback_requested
+        .where(
+          application_form: {
+            recruitment_cycle_year: RECRUITMENT_CYCLE_YEAR,
+            phase: PHASE,
+          },
+        )
+        .where(application_form: { id: ApplicationChoice
+            .where(status: 'unsubmitted')
+            .select('application_form_id') })
```

Comment:
At first I'd like to but because this is a data migration after next Tuesday we won't need that anymore so probably the data migration will be deleted after?